Create Add_UserAgent_in_comment use case (#754)
SEKOIA-IO/documentation: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md

# Enrich Alert with User agent in comment

> This use case describes how to enrich an alert with the User agent in comment

Reviewer: ```suggestion
This use case describes how to enrich the comments of an alert with the User agent extracted from the associated events.
```

Reviewer: A suggestion, it's up to you. However, if you keep your sentence, you should add the final dot.

## Prerequisites

- A SEKOIA.IO XDR licence
- The user that configures the Playbook should have a role that contains:
	* The SYMPHONY permissions
- An API Key with a role that contains at least the following permission:
	* `SIC_READ_ALERTS`
	* `SIC_WRITE_ALERTS_COMMENT`

!!! note
    To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).

## Playbook configuration

Find the playbook configuration below: 

> ![Playbook Add_UserAgent_in_comment](docs/assets/playbooks/library/UseCases/Add_UserAgent_in_comment.md.png)

Reviewer: The image is not displayed in the preview, is it normal?

| Module | Configuration |
| --- | --- |

> | Alert Created | configure module & trigger configuration |
> | Get_an_alert | uuid = `alert_uuid` of **Created alert** |

Reviewer: ```suggestion
| Alert Created | configure module & trigger configuration |
| Get Alert | uuid = `alert_uuid` of **Created alert** |
```

| Get Event Field Common Values | See fields below |

> | post_a_comment_on_an_alert | See fields below |

Reviewer: ```suggestion
| Comment Alert | See fields below |
```

`Get Event Field Common Values`
  - **earliest_time**   `{{ ((node.2.first_seen_at | iso8601_to_timestamp) - 3600000) | timestamp_to_iso8601 }}`
  - **fields**  `user_agent.original`
  - **latest_time**    `now`    

>   - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND [entity.name](http://entity.name/):"{{ node.2['entity']['name'] }}"`

Reviewer: ```suggestion
  - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND entity.name:"{{ node.2['entity']['name'] }}"`
```

Reviewer: I think there is a wrong copy/paste in the query

>   - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND [entity.name](http://entity.name/):"{{ node.2['entity']['name'] }}"`

Reviewer: Are you sure that this query allows in all case to retrieve events that contain a User agent? If not, you should specify that the query could vary according to the event types / intakes

> `post_a_comment_on_an_alert`

Reviewer: The format is not valid in the preview, you should apply changes

Author: correction done on entity.name
